src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Commands.cs

```diff
@@ -53,7 +53,7 @@ public static partial class IViewModelExtensions
 		/// <param name="viewModel">This <see cref="IViewModel"/>.</param>
 		/// <param name="execute">The command's execution delegate.</param>
 		/// <param name="configure">The optional function to configure the command builder.</param>
-		/// <param name="name">The command name.</param>
+		/// <param name="name">The optional command name.</param>
```

Comment:
All parameters marked with `[CallerMemberName]` are not optional.
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.callermembernameattribute?view=net-5.0